Symfony/CS/Tests/Fixer/Symfony/MultilineArrayTrailingCommaFixerTest.php

```diff
@@ -298,7 +260,78 @@ function foo(array $a)
         "string"
         /* foo */);',
             ),
-            array(
+        );
+    }
+
+    /**
+     * @dataProvider provideExamples54
+     * @requires PHP 5.4
+     */
+    public function testFix54($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideExamples54()
+    {
+        return array(
+                // short syntax tests
+                array('<?php $x = array([]);'),
+                array('<?php $x = [[]];'),
```

Comment:
yea... while on that topic, are you sure this is the way to go? For example the short open tag fixers work just fine even when running without short tags being enabled..